macos/Sources/Ghostty/Ghostty.App.swift

```diff
@@ -364,14 +373,20 @@ extension Ghostty {
         static func writeClipboard(_ userdata: UnsafeMutableRawPointer?, string: UnsafePointer<CChar>?, location: ghostty_clipboard_e, confirm: Bool) {
             let surface = self.surfaceUserdata(from: userdata)
 
-            // We only support the standard clipboard
-            if (location != GHOSTTY_CLIPBOARD_STANDARD) { return }
+            let pasteboard: NSPasteboard
+            switch location {
+            case GHOSTTY_CLIPBOARD_STANDARD:
+                pasteboard = NSPasteboard.general
+            case GHOSTTY_CLIPBOARD_SELECTION:
+                pasteboard = selectionPasteboard
+            default:
+                return
+            }
```

Comment:
I think you could assign the `pasteboard` to the result of the `switch` expression to make this a little more concise.